ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py

```diff
@@ -68,6 +68,41 @@ def _get_axis_and_p(draw, kind="valid"):
 # ------------ #
 
 
+# mode
+@handle_frontend_test(
+    fn_tree="torch.mode",
+    dtype_input_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=1,
+        valid_axis=True,
+        force_int_axis=True,
+    ),
+    keepdim=st.booleans(),
+)
+def test_ivy_mode(
+    *,
+    dtype_input_axis,
```

Comment:
the tests are failing unfortunately, could you please run them with the command: `python3 -m pytest ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py::test_ivy_mode` and make sure they are all passing? 
A small tip: you might want to run the torch function natively to observe some behaviours and make sure the output in the ivy frontend behaves in the same way, thanks!

Reply:
Hey, @juliagsy can you text me over Discord I'm getting some errors using this command. Thanks!

Reply:
Hey! Could you please comment them here so that I don't mess up message from other PRs? Thanks!

Reply:
You may also try running tests with the command `pytest ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py::test_ivy_mode`, whichever works

Reply:
key error : elementwise

Reply:
hey, could you please share the error log you see so that it's easier to know where the error comes from?
also, there's usually traceback where you can see which lines of code it went through, which is helpful for resolving errors, thanks!

Reply:
_________________ ERROR collecting ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py _________________
ivy\utils\_importlib.py:65: in _from_import
    mod_globals[alias] = module.__dict__[entry_name]
E   KeyError: 'elementwise'

During handling of the above exception, another exception occurred:
C:\Python311\Lib\site-packages\_pytest\runner.py:341: in from_call
    result: Optional[TResult] = func()
C:\Python311\Lib\site-packages\_pytest\runner.py:372: in <lambda>
    call = CallInfo.from_call(lambda: list(collector.collect()), "collect")
C:\Python311\Lib\site-packages\_pytest\python.py:531: in collect
    self._inject_setup_module_fixture()
C:\Python311\Lib\site-packages\_pytest\python.py:545: in _inject_setup_module_fixture
    self.obj, ("setUpModule", "setup_module")
C:\Python311\Lib\site-packages\_pytest\python.py:310: in obj
    self._obj = obj = self._getobj()
C:\Python311\Lib\site-packages\_pytest\python.py:528: in _getobj
    return self._importtestmodule()
C:\Python311\Lib\site-packages\_pytest\python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
C:\Python311\Lib\site-packages\_pytest\pathlib.py:567: in import_path
    importlib.import_module(module_name)
C:\Python311\Lib\importlib\__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1206: in _gcd_import
    ???
<frozen importlib._bootstrap>:1178: in _find_and_load
    ???
<frozen importlib._bootstrap>:1149: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:690: in _load_unlocked
    ???
C:\Python311\Lib\site-packages\_pytest\assertion\rewrite.py:178: in exec_module
    exec(co, module.__dict__)
ivy_tests\test_ivy\test_frontends\test_torch\test_reduction_ops.py:8: in <module>
    from ivy_tests.test_ivy.test_functional.test_core.test_statistical import (
<frozen importlib._bootstrap>:1178: in _find_and_load
    ???
<frozen importlib._bootstrap>:1149: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:690: in _load_unlocked
    ???
C:\Python311\Lib\site-packages\_pytest\assertion\rewrite.py:178: in exec_module
    exec(co, module.__dict__)
ivy_tests\test_ivy\test_functional\test_core\test_statistical.py:88: in <module>
    @handle_test(
ivy_tests\test_ivy\helpers\testing_helpers.py:421: in test_wrapper
    supported_device_dtypes = _get_supported_devices_dtypes(fn_name, fn_mod)
ivy_tests\test_ivy\helpers\testing_helpers.py:313: in _get_supported_devices_dtypes
    supported_device_dtypes[backend_str] = _get_supported_devices_dtypes_helper(
ivy_tests\test_ivy\helpers\testing_helpers.py:259: in _get_supported_devices_dtypes_helper
    with BackendHandler.update_backend(backend_str) as backend:
ivy_tests\test_ivy\helpers\pipeline_helper.py:18: in __enter__
    return ivy.with_backend(self.backend)
ivy\utils\backend\handler.py:65: in _prevent_access_locally
    return fn(*args, **kwargs)
ivy\utils\backend\handler.py:591: in with_backend
    backend_module = _importlib._import_module(
ivy\utils\_importlib.py:126: in _import_module
    spec.loader.exec_module(module)
ivy\utils\backend\ast_helpers.py:283: in exec_module
    raise e
ivy\utils\backend\ast_helpers.py:279: in exec_module
    exec(compiled_obj, module.__dict__)
ivy\functional\backends\tensorflow\__init__.py:1: in <module>
    ???
ivy\utils\_importlib.py:72: in _from_import
    mod_globals[alias] = _import_module(in_name, package)
ivy\utils\_importlib.py:126: in _import_module
    spec.loader.exec_module(module)
ivy\utils\backend\ast_helpers.py:283: in exec_module
    raise e
ivy\utils\backend\ast_helpers.py:279: in exec_module
    exec(compiled_obj, module.__dict__)
ivy\functional\backends\tensorflow\elementwise.py:4: in <module>
    import tensorflow_probability as tfp
C:\Python311\Lib\site-packages\tensorflow_probability\__init__.py:20: in <module>
    from tensorflow_probability import substrates
C:\Python311\Lib\site-packages\tensorflow_probability\substrates\__init__.py:17: in <module>
    from tensorflow_probability.python.internal import all_util
C:\Python311\Lib\site-packages\tensorflow_probability\python\__init__.py:138: in <module>
    dir(globals()[pkg_name])  # Forces loading the package from its lazy loader.
C:\Python311\Lib\site-packages\tensorflow_probability\python\internal\lazy_loader.py:57: in __dir__
    module = self._load()
C:\Python311\Lib\site-packages\tensorflow_probability\python\internal\lazy_loader.py:40: in _load
    module = importlib.import_module(self.__name__)
C:\Python311\Lib\importlib\__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
C:\Python311\Lib\site-packages\tensorflow_probability\python\experimental\__init__.py:31: in <module>
    from tensorflow_probability.python.experimental import bayesopt
C:\Python311\Lib\site-packages\tensorflow_probability\python\experimental\bayesopt\__init__.py:17: in <module>
    from tensorflow_probability.python.experimental.bayesopt import acquisition
C:\Python311\Lib\site-packages\tensorflow_probability\python\experimental\bayesopt\acquisition\__init__.py:17: in <module>
    from tensorflow_probability.python.experimental.bayesopt.acquisition.acquisition_function import AcquisitionFunction
C:\Python311\Lib\site-packages\tensorflow_probability\python\experimental\bayesopt\acquisition\acquisition_function.py:22: in <module>
    from tensorflow_probability.python.internal import prefer_static as ps
C:\Python311\Lib\site-packages\tensorflow_probability\python\internal\prefer_static.py:361: in <module>
    ones_like = _copy_docstring(tf.ones_like, _ones_like)
C:\Python311\Lib\site-packages\tensorflow_probability\python\internal\prefer_static.py:84: in _copy_docstring
    raise ValueError(
E   ValueError: Arg specs do not match: original=FullArgSpec(args=['input', 'dtype', 'name', 'layout'], varargs=None, varkw=None, defaults=(None, None, None), kwonlyargs=[], kwonlydefaults=None, annotations={}), new=FullArgSpec(args=['input', 'dtype', 'name'], varargs=None, varkw=None, defaults=(None, None), kwonlyargs=[], kwonlydefaults=None, annotations={}), fn=<function ones_like_v2 at 0x00000261C4965F80>
--------------------------------------------------- Captured stdout ---------------------------------------------------

Arg specs do not match: original=FullArgSpec(args=['input', 'dtype', 'name', 'layout'], varargs=None, varkw=None, defaults=(None, None, None), kwonlyargs=[], kwonlydefaults=None, annotations={}), new=FullArgSpec(args=['input', 'dtype', 'name'], varargs=None, varkw=None, defaults=(None, None), kwonlyargs=[], kwonlydefaults=None, annotations={}), fn=<function ones_like_v2 at 0x00000261C4965F80>
Arg specs do not match: original=FullArgSpec(args=['input', 'dtype', 'name', 'layout'], varargs=None, varkw=None, defaults=(None, None, None), kwonlyargs=[], kwonlydefaults=None, annotations={}), new=FullArgSpec(args=['input', 'dtype', 'name'], varargs=None, varkw=None, defaults=(None, None), kwonlyargs=[], kwonlydefaults=None, annotations={}), fn=<function ones_like_v2 at 0x00000261C4965F80>
--------------------------------------------------- Captured stderr ---------------------------------------------------
WARNING:root:   Some binaries seem to be missing in your system. This could be either because we don't have compatible binaries for your system or that newer binaries were available. In the latter case, calling ivy.utils.cleanup_and_fetch_binaries() should fetch the binaries binaries. Feel free to create an issue on https://github.com/unifyai/ivy.git in case of the former

WARNING:root:
Following are the supported configurations :
compiler : cp38-none-manylinux_2_17_x86_64, cp310-none-manylinux_2_17_x86_64
engines : cp310-none-manylinux_2_17_x86_64

WARNING:root:   ivy\compiler\_compiler.so not found.
WARNING:root:   ivy\compiler\utils\CC.so not found.
WARNING:root:   ivy\compiler\utils\CD.so not found.
WARNING:root:   ivy\compiler\utils\CI.so not found.
WARNING:root:   ivy\compiler\utils\CL.so not found.
WARNING:root:   ivy\compiler\utils\CM.so not found.
WARNING:root:   ivy\compiler\utils\CV.so not found.
WARNING:root:   ivy\compiler\utils\CX.so not found.
WARNING:root:   ivy\compiler\utils\C.so not found.
WARNING:root:   ivy\compiler\utils\DC.so not found.
WARNING:root:   ivy\compiler\utils\DD.so not found.
WARNING:root:   ivy\compiler\utils\DI.so not found.
WARNING:root:   ivy\compiler\utils\DL.so not found.
WARNING:root:   ivy\compiler\utils\DM.so not found.
WARNING:root:   ivy\compiler\utils\DV.so not found.
WARNING:root:   ivy\compiler\utils\DX.so not found.
WARNING:root:   ivy\compiler\utils\D.so not found.
WARNING:root:   ivy\compiler\utils\IC.so not found.
WARNING:root:   ivy\compiler\utils\ID.so not found.
WARNING:root:   ivy\compiler\utils\II.so not found.
WARNING:root:   ivy\compiler\utils\IL.so not found.
WARNING:root:   ivy\compiler\utils\IM.so not found.
WARNING:root:   ivy\compiler\utils\IV.so not found.
WARNING:root:   ivy\compiler\utils\IX.so not found.
WARNING:root:   ivy\compiler\utils\I.so not found.
WARNING:root:   ivy\compiler\utils\LC.so not found.
WARNING:root:   ivy\compiler\utils\LD.so not found.
WARNING:root:   ivy\compiler\utils\LI.so not found.
WARNING:root:   ivy\compiler\utils\LL.so not found.
WARNING:root:   ivy\compiler\utils\LM.so not found.
WARNING:root:   ivy\compiler\utils\LV.so not found.
WARNING:root:   ivy\compiler\utils\LX.so not found.
WARNING:root:   ivy\compiler\utils\L.so not found.
WARNING:root:   ivy\compiler\utils\MI.so not found.
WARNING:root:   ivy\compiler\utils\ML.so not found.
WARNING:root:   ivy\compiler\utils\MV.so not found.
WARNING:root:   ivy\compiler\utils\MX.so not found.
WARNING:root:   ivy\compiler\utils\M.so not found.
WARNING:root:   ivy\compiler\utils\VC.so not found.
WARNING:root:   ivy\compiler\utils\VD.so not found.
WARNING:root:   ivy\compiler\utils\VI.so not found.
WARNING:root:   ivy\compiler\utils\VL.so not found.
WARNING:root:   ivy\compiler\utils\VM.so not found.
WARNING:root:   ivy\compiler\utils\VV.so not found.
WARNING:root:   ivy\compiler\utils\VX.so not found.
WARNING:root:   ivy\compiler\utils\V.so not found.
WARNING:root:   ivy\compiler\utils\XC.so not found.
WARNING:root:   ivy\compiler\utils\XD.so not found.
WARNING:root:   ivy\compiler\utils\XI.so not found.
WARNING:root:   ivy\compiler\utils\XL.so not found.
WARNING:root:   ivy\compiler\utils\XM.so not found.
WARNING:root:   ivy\compiler\utils\XV.so not found.
WARNING:root:   ivy\compiler\utils\XX.so not found.
WARNING:root:   ivy\compiler\utils\X.so not found.
WARNING:root:   ivy\engines\ivy2xla.so not found.
WARNING:root:   ivy\engines\XLA\rust_api\python_frontend\activations.so not found.
WARNING:root:   ivy\engines\XLA\rust_api\python_frontend\creation.so not found.
WARNING:root:   ivy\engines\XLA\rust_api\python_frontend\elementwise.so not found.
WARNING:root:   ivy\engines\XLA\rust_api\python_frontend\general.so not found.
WARNING:root:   ivy\engines\XLA\rust_api\python_frontend\layers.so not found.
WARNING:root:   ivy\engines\XLA\rust_api\python_frontend\linear_algebra.so not found.
WARNING:root:   ivy\engines\XLA\rust_api\python_frontend\manipulation.so not found.
WARNING:root:   ivy\engines\XLA\rust_api\python_frontend\norms.so not found.
WARNING:root:   ivy\engines\XLA\rust_api\python_frontend\stateful_activations.so not found.
WARNING:root:   ivy\engines\XLA\rust_api\python_frontend\stateful_layers.so not found.
WARNING:root:   ivy\engines\XLA\rust_api\python_frontend\stateful_norms.so not found.
WARNING:root:   ivy\engines\XLA\rust_api\python_frontend\statistical.so not found.
WARNING:root:   ivy\engines\XLA\rust_api\python_frontend\xla_core.so not found.
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1697798946.348682    7960 tfrt_cpu_pjrt_client.cc:349] TfrtCpuClient created.
WARNING:jax._src.xla_bridge:No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
Traceback (most recent call last):
  File "C:\Users\kurel\Downloads\ivy\ivy\utils\_importlib.py", line 65, in _from_import
    mod_globals[alias] = module.__dict__[entry_name]
                         ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: 'elementwise'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\kurel\Downloads\ivy\ivy\utils\backend\ast_helpers.py", line 279, in exec_module
    exec(compiled_obj, module.__dict__)
  File "C:\Users\kurel\Downloads\ivy\ivy\functional\backends\tensorflow\elementwise.py", line 4, in <module>
    import tensorflow_probability as tfp
  File "C:\Python311\Lib\site-packages\tensorflow_probability\__init__.py", line 20, in <module>
    from tensorflow_probability import substrates
  File "C:\Python311\Lib\site-packages\tensorflow_probability\substrates\__init__.py", line 17, in <module>
    from tensorflow_probability.python.internal import all_util
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\__init__.py", line 138, in <module>
    dir(globals()[pkg_name])  # Forces loading the package from its lazy loader.
    ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\internal\lazy_loader.py", line 57, in __dir__
    module = self._load()
             ^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\internal\lazy_loader.py", line 40, in _load
    module = importlib.import_module(self.__name__)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\experimental\__init__.py", line 31, in <module>
    from tensorflow_probability.python.experimental import bayesopt
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\experimental\bayesopt\__init__.py", line 17, in <module>
    from tensorflow_probability.python.experimental.bayesopt import acquisition
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\experimental\bayesopt\acquisition\__init__.py", line 17, in <module>
    from tensorflow_probability.python.experimental.bayesopt.acquisition.acquisition_function import AcquisitionFunction
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\experimental\bayesopt\acquisition\acquisition_function.py", line 22, in <module>
    from tensorflow_probability.python.internal import prefer_static as ps
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\internal\prefer_static.py", line 361, in <module>
    ones_like = _copy_docstring(tf.ones_like, _ones_like)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\internal\prefer_static.py", line 84, in _copy_docstring
    raise ValueError(
ValueError: Arg specs do not match: original=FullArgSpec(args=['input', 'dtype', 'name', 'layout'], varargs=None, varkw=None, defaults=(None, None, None), kwonlyargs=[], kwonlydefaults=None, annotations={}), new=FullArgSpec(args=['input', 'dtype', 'name'], varargs=None, varkw=None, defaults=(None, None), kwonlyargs=[], kwonlydefaults=None, annotations={}), fn=<function ones_like_v2 at 0x00000261C4965F80>
Traceback (most recent call last):
  File "C:\Users\kurel\Downloads\ivy\ivy\utils\_importlib.py", line 65, in _from_import
    mod_globals[alias] = module.__dict__[entry_name]
                         ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: 'elementwise'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\kurel\Downloads\ivy\ivy\utils\backend\ast_helpers.py", line 279, in exec_module
    exec(compiled_obj, module.__dict__)
  File "C:\Users\kurel\Downloads\ivy\ivy\functional\backends\tensorflow\__init__.py", line 1, in <module>
    # global

  File "C:\Users\kurel\Downloads\ivy\ivy\utils\_importlib.py", line 72, in _from_import
    mod_globals[alias] = _import_module(in_name, package)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\kurel\Downloads\ivy\ivy\utils\_importlib.py", line 126, in _import_module
    spec.loader.exec_module(module)
  File "C:\Users\kurel\Downloads\ivy\ivy\utils\backend\ast_helpers.py", line 283, in exec_module
    raise e
  File "C:\Users\kurel\Downloads\ivy\ivy\utils\backend\ast_helpers.py", line 279, in exec_module
    exec(compiled_obj, module.__dict__)
  File "C:\Users\kurel\Downloads\ivy\ivy\functional\backends\tensorflow\elementwise.py", line 4, in <module>
    import tensorflow_probability as tfp
  File "C:\Python311\Lib\site-packages\tensorflow_probability\__init__.py", line 20, in <module>
    from tensorflow_probability import substrates
  File "C:\Python311\Lib\site-packages\tensorflow_probability\substrates\__init__.py", line 17, in <module>
    from tensorflow_probability.python.internal import all_util
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\__init__.py", line 138, in <module>
    dir(globals()[pkg_name])  # Forces loading the package from its lazy loader.
    ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\internal\lazy_loader.py", line 57, in __dir__
    module = self._load()
             ^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\internal\lazy_loader.py", line 40, in _load
    module = importlib.import_module(self.__name__)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\experimental\__init__.py", line 31, in <module>
    from tensorflow_probability.python.experimental import bayesopt
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\experimental\bayesopt\__init__.py", line 17, in <module>
    from tensorflow_probability.python.experimental.bayesopt import acquisition
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\experimental\bayesopt\acquisition\__init__.py", line 17, in <module>
    from tensorflow_probability.python.experimental.bayesopt.acquisition.acquisition_function import AcquisitionFunction
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\experimental\bayesopt\acquisition\acquisition_function.py", line 22, in <module>
    from tensorflow_probability.python.internal import prefer_static as ps
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\internal\prefer_static.py", line 361, in <module>
    ones_like = _copy_docstring(tf.ones_like, _ones_like)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\tensorflow_probability\python\internal\prefer_static.py", line 84, in _copy_docstring
    raise ValueError(
ValueError: Arg specs do not match: original=FullArgSpec(args=['input', 'dtype', 'name', 'layout'], varargs=None, varkw=None, defaults=(None, None, None), kwonlyargs=[], kwonlydefaults=None, annotations={}), new=FullArgSpec(args=['input', 'dtype', 'name'], varargs=None, varkw=None, defaults=(None, None), kwonlyargs=[], kwonlydefaults=None, annotations={}), fn=<function ones_like_v2 at 0x00000261C4965F80>
================================================== warnings summary ===================================================
ivy\utils\exceptions.py:390
  C:\Users\kurel\Downloads\ivy\ivy\utils\exceptions.py:390: UserWarning: The current backend: 'jax' does not support inplace updates natively. Ivy would quietly create new arrays when using inplace updates with this backend, leading to memory overhead (same applies for views). If you want to control your memory management, consider doing ivy.set_inplace_mode('strict') which should raise an error whenever an inplace update is attempted with this backend.
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=============================================== short test summary info ===============================================
ERROR ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py - ValueError: Arg specs do not match: original=FullArgSpec(args=['input', 'dtype', 'name', 'layout'], varargs=None, v...
============================================= 1 warning, 1 error in 8.11s =============================================
ERROR: not found: C:\Users\kurel\Downloads\ivy\ivy_tests\test_ivy\test_frontends\test_torch\test_reduction_ops.py::test_ivy_mode
(no name 'C:\\Users\\kurel\\Downloads\\ivy\\ivy_tests\\test_ivy\\test_frontends\\test_torch\\test_reduction_ops.py::test_ivy_mode' in any of [<Module test_reduction_ops.py>])

Hey, The error log can be found above. Thanks!

Reply:
Hey! The test function named changed, so you should run `python3 -m pytest ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py::test_torch_mode` instead
The error is about not being able to find `test_ivy_mode`

Reply:
FAILED ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py::test_torch_mode[cpu-numpy-False-False] - hypothesis.errors.InvalidArgument: Cannot sample from a length-zero sequence.
FAILED ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py::test_torch_mode[cpu-torch-False-False] - KeyError: 'cpu'
Hey, after running the above it throws the above errors. Thanks!